bin/new.js

```diff
@@ -0,0 +1,23 @@
+var Chalk = require('chalk');
+var Cli = require('structured-cli');
+var Open = require('open');
+
+
+module.exports = Cli.createCommand('new', {
```

Comment:
I think `wt new` going to be confusing against `wt create`. Could we instead add a flag to `wt edit`, allow `wt edit` to be called without parameters, or auto-detect if the `wt edit {name}` exists or not?